src/user/using-the-aggregate-data-entry-app.md

```diff
@@ -0,0 +1,190 @@
+| ![](resources/images/data_entry_beta/cell-synced.png)   | The cell value is saved on the server.                                                                                                                |
+| ![](resources/images/data_entry_beta/cell-waiting.png) | The cell value is saved locally and is syncing, or waiting to sync, to the server.                                                                                              |
+| ![](resources/images/data_entry_beta/cell-invalid.png)     | There's a problem with the cell value. Click or hover over the cell to learn more about the problem. These invalid values are not saved to the server or locally. |
+| ![](resources/images/data_entry_beta/cell-comment.png) | The value has a comment.                                                                                                                         |
+| ![](resources/images/data_entry_beta/cell-readonly.png)  | The cell is locked and the value can't be edited.                                                                                                                                                 |
+
+### Filtering a form { #aggregate_data_entry_app.filtering }
+Filtering is useful if you're looking for a certain cell in a form. You can filter the whole form, filter single sections, or both. Any cells that don't match the filter will be hidden.
+
+#### Filtering the whole form form { #aggregate_data_entry_app.filtering_whole_form }
+
+![](resources/images/data_entry_beta/filter-form.png) { width=60% }
+
+To filter the whole data entry form, enter a value into the input at the top of the form.
+
+#### Filtering a section { #aggregate_data_entry_app.filtering_section }
+
+![](resources/images/data_entry_beta/filter-section.png) { width=60% }
+
+For section form you can filter only inside a single section. Enter a value into the input at the top of a section.
+
+### Validation { #aggregate_data_entry_app.validation }
+When you're done entering data you can run validation on the data values. Validation checks the values against rules set up by your DHIS2 instance. 
+
+To run validation, click the *Run validation* button in the bottom bar. 
+
+![](resources/images/data_entry_beta/validation-button.png) { width=60% }
+
+The validation results are shown in the details sidebar, grouped into high, medium, and low priority results. Once you've fixed validation issues, click the *Run validation again* button to recheck data values.
+
+![](resources/images/data_entry_beta/validation-results.png) 
+
+
+### Completing a form { #aggregate_data_entry_app.completing }
+
+After entering data and running validation, the last step is completing a form. Mark a form as complete by clicking the *Mark complete* button in the bottom bar.
```

Comment:
```suggestion
After entering data and running validation, the last step is completing a form. This indicates that all the intended data has been entered, and that empty cells are supposed to be empty. Mark a form as complete by clicking the *Mark complete* button in the bottom bar.
```

Reply:
Thanks, committed something along these lines, just changed the wording slightly.